src/__tests__/components/forms/CreateSmsTemplate/CreateSmsTemplate.test.tsx

```diff
@@ -110,7 +110,7 @@ describe('CreateSmsTemplate component', () => {
       throw new Error('Template name box not found');
     }
 
-    const longMessage = 'x'.repeat(300);
+    const longMessage = 'x'.repeat(30);
```

Comment:
30 chars is not a particularly long message, was this a deliberate change?

Reply:
My PC was timing out doing the 300 characters and the focus of the unit test is that the character count component works - which can still be done with 30

Reply:
Seems like there might be a more fundamental issue with the test if it times out.  Looks like we type then immediately query the page which may not have updated yet.  Reducing this to 30 does not seem like the right solution.  Maybe better to skip the test and add it into CCM-6950 to fix.

Reply:
i don't think it's an issue with the test as it always passes on CI, it was just failing for me. not sure it's worth a specific ticket to look into something that only affects my PC. we can move it back to 300 if you feel this makes a significant difference to the test

Reply:
changed back to 300